.circleci/config.yml

```diff
@@ -269,10 +270,16 @@ jobs:
       - run: *install_cached_packages
       - attach_workspace:
           at: .
-      - run: cover -write cover_db cover_db*
-      - run: cover -select_re '^(lib|script|t)/' -report html_minimal
+      - run:
+          name: Merge coverage databases
+          command: |
+            cover -write cover_db cover_db*
+            ls cover_db/*cov*
+      - run: make coverage-html
       - store_artifacts: *store_cover_db
-      - run: cover -select_re '^(lib|script|t)/' -report codecov
+      - run: sudo cpanm -nq 'Devel::Cover::Report::Codecovbash'
```

Comment:
Please also ensure that the package is submitted to openSUSE:Factory. This can be done as part of the ticket, pending acceptance not blocking the PR.